source/extensions/filters/common/rbac/engine.h

```diff
@@ -25,9 +25,11 @@ class RoleBasedAccessControlEngine {
    * @param headers    the headers of the incoming request used to identify the action/principal. An
    *                   empty map should be used if there are no headers available.
    * @param metadata   the metadata with additional information about the action/principal.
+   * @param effective_policy_id  the matching policy's ID to identity the source of the allow/deny.
```

Comment:
Can you specify that this will be filled with the policy ID? It sounds like it's an input parameter.

Reply:
updated comment; thank you for the review !